frontend/src/components/campAdmin/CampActivityProgressLabels.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <content-group>
+    <slot name="title">
+      <div class="ec-content-group__title py-1 subtitle-1">
+        {{ $tc('components.campAdmin.campActivityProgressLabels.title') }}
+        <dialog-activity-progress-label-create v-if="!disabled" :camp="camp()">
+          <template #activator="{ on }">
+            <button-add
+              color="secondary"
+              text
+              :hide-label="$vuetify.breakpoint.xsOnly"
+              class="my-n2"
+              v-on="on"
+            >
+              {{ $tc('components.campAdmin.campActivityProgressLabels.create') }}
+            </button-add>
+          </template>
+        </dialog-activity-progress-label-create>
+      </div>
+    </slot>
+    <v-skeleton-loader v-if="loading" type="article" />
+    <v-list>
+      <v-list-item
+        v-for="(progressLabel, idx) in progressLabels"
+        :key="progressLabel._meta.self"
+        class="px-0"
+      >
+        <v-list-item-content class="pt-0 pb-2">
+          <v-list-item-title>
+            {{ idx + 1 }}) {{ progressLabel.title }}
```

Comment:
I would leave the numbering out. It might confuse the users such that they try to add numbering to the title themselves, or some users might not use labels in an "ordered" way at all. Also, the 1) 2) 3) numbering style might not be normal depending on the locale.
```suggestion
            {{ progressLabel.title }}
```